blog/2020-01-everything-github/README.md

```diff
@@ -0,0 +1,713 @@
+            ####
+            npm run ng -- deploy --base-href=/everything-github-demo/ --name="Displayed Username" --email=your.mail@example.org
+    ```
+3. Fortunately, the step `actions/checkout@v2` checks-out a nice git repository, where the remote `origin` is already adjusted.
+    But username and email are not set.
+    Therefore we have to include these two values via parameters.
+    Make sure to replace **"Displayed Username"** (e.g. "I am a bot") and **your.mail@example.org** with meaningful values in the above example.
+    At this place you do not have to specify any real name and mail.
+    But you don't want to overdo it, because the provided values later will be shown in the git history.
+
+4. If you want Github Actions to perform unit tests and end-to-end tests, you will need to make some small additional configurations in your Angular app.
+    The necessary changes are described in the [Angular CLI wiki](https://github.com/angular/angular-cli/wiki/stories-continuous-integration#continuous-integration).
+    To be precise:
+
+    1. You need to [update the test configurations for Karma and Protractor](https://github.com/angular/angular-cli/wiki/stories-continuous-integration#update-test-configuration)
+    2. You have to setup a specific [chrome driver version](https://github.com/angular/angular-cli/wiki/stories-continuous-integration#chromedriver).
+       Otherwise, the e2e tests will fail with an error message like this:
+      `... E/launcher - session not created: This version of ChromeDriver only supports Chrome version 79`
+
+    After those changes have been done, you can un-comment sections for the tests in above example.
+
+5. We can also control when our workflows are triggered:
+    It can be helpful to not have our workflow run on every push to every branch in the repo.
+     - For example, this workflow only runs on push events to `master` and `release` branches:
+
+        ```yml
+        on:
+          push:
+            branches:
+            - master
+            - release/*  # branches matching refs/heads/release/*
+        ```
+      
+        Here we see the very common convention of grouping branches with a slash (e.g. `relase/42`).
+        Many git clients will display those groups like folders.
+
+     - As another example, we can run the script every day of the week from Monday - Friday at 02:00:
+
+        ```yml
+        on:
+          schedule:
+            - cron: 0 2 * * 1-5
+        ```
+
+   - For more information see [Events that trigger workflows](https://help.github.com/articles/events-that-trigger-workflows) and [Workflow syntax for GitHub Actions](https://help.github.com/articles/workflow-syntax-for-github-actions#on).
+
+6. Then, click on **Start commit**, add message and description if you like and click on **Commit new file**.
+
+    ![start commit](./screenshot_5_start-commit.png)
+
+4.  **Done!** 🚀.  
+   The action will run by itself the first time.
```

Comment:
If you do this [change](https://github.com/angular-schule/website-articles/pull/7/files#r364254872), you can remove this line:
```suggestion
```